Berlin/bars/spektrum.md

```diff
@@ -0,0 +1,15 @@
+# kaufbar
+
```

Comment:
Oops! That must've been a copypasta error. 
Let me just quickly amend this commit.